extensions/websockets-next/spi/src/main/java/io/quarkus/websockets/next/runtime/spi/telemetry/WebSocketEndpointContext.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.websockets.next.runtime.spi.telemetry;
+
+import java.util.Map;
+
+public record WebSocketEndpointContext(String path, EndpointKind endpointKind, Map<String, Object> connectionContextStorage,
```

Comment:
It might make sense to add some javadoc for this record (given the fact that it's a part of a public SPI).

Reply:
sorry @mkouba , completely missed this comment, I'll add some javadoc

Reply:
Done.